From: Arnaud Rebillout <arnaud.rebillout@collabora.com>
Date: Fri, 18 Jan 2019 18:54:15 +0700
Subject: Disable containerizedengine/update_test.go

This test FTBFS, see <https://github.com/docker/cli/pull/1561>.
Please re-enable this test when this MR is accepted.

Origin: vendor, Debian
Forwarded: not-needed, Debian-specific
Signed-off-by: Arnaud Rebillout <arnaud.rebillout@collabora.com>

--- a/cli/internal/containerizedengine/update_test.go
+++ b/cli/internal/containerizedengine/update_test.go
@@ -1,3 +1,5 @@
+// +build ignore
+
 package containerizedengine
 
 import (
